### PR TITLE
feat: Implement util functions to sort scenario lists

### PR DESCRIPTION
**Pull request by COSMO-VMI**

### CODE REVIEW

File: src/utils/SortScenarioListUtils.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+function indexOfById (list, id) {
+  for (let i = 0; i < list.length; ++i) {
+    const item = list[i];
+    if (item.id === id) {
+      return i;
+    }
+  }
+  return -1;
+}
```
Comment (csm-thu) [inline src/utils/SortScenarioListUtils.js:18]: This function can be replaced by the **find** method of arrays:
`list.find(item => item.id === id)`
If no items match the expected condition, then _undefined_ is returned.
More info here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find

**Review by csm-thu — APPROVED** [review]
LGTM

File: src/utils/SortScenarioListUtils.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+// SORT
+export const testSort = (scenarioList) => {
+  const sortedList = sortScenarioList(scenarioList.data.slice());
+  console.log(sortedList);
+};
```
[the remark this answers is not in the document]
Comment (COSMO-VMI) [reply src/utils/SortScenarioListUtils.js:8]: Nope, thanks

File: src/utils/SortScenarioListUtils.js (hunk 1 above)
Comment (COSMO-VMI) [reply src/utils/SortScenarioListUtils.js:18]: Nice, I don't know all the these algo yet. I'll update it.

Comment (COSMO-VMI) [reply src/utils/SortScenarioListUtils.js:18]: Also, it's more `findIndex()` than `find()`

**Review by Nikeleos — APPROVED** [review]

File: src/utils/SortScenarioListUtils.js (hunk 1 above)
Comment (jreynard-code) [reply src/utils/SortScenarioListUtils.js:12]: don't forget to remove the function if it's not used ;)

**Review by jreynard-code — CHANGES_REQUESTED** [review]
LGTM, just remove the unused function ;)

**Pull request merged**